src/main/java/org/opentripplanner/api/mapping/FareMapper.java

```diff
@@ -14,9 +15,8 @@
 import org.opentripplanner.routing.core.FareComponent;
 import org.opentripplanner.routing.core.Money;
 
-public class FareMapper {
-
-  public static ApiFare mapFare(Fare fare) {
+public record FareMapper(Locale locale) {
```

Comment:
Please use records for what they are ment for - "plain data carriers". You save a few lines of code here, but it is confusing - takes longer to read when used in an unconventional way.

Reply:
Done